index.html

```diff
@@ -0,0 +1,203 @@
+                    </p>
+                </div>
+
+                <div class="bg-white/10 p-4 flex md:flex-col justify-center items-center text-center">
+                    <i class="hover:text-[#f28f10] fa-solid fa-handshake fa-2x md:fa-3x"></i>
+                    <h4 class="text-[#f28f10] hover:text-white  text-xl px-2 py-2 ">CONNECT</h4>
+                    <p>
+                        Connect with people across the globe and learn from them.
+                    </p>
+                </div>
+
+                <div class="bg-white/10 p-4 flex md:flex-col justify-center items-center text-center">
+                    <i class="hover:text-[#f28f10] fa-solid fa-earth-americas fa-2x md:fa-3x"></i>
+                    <h4 class="text-[#f28f10] hover:text-white text-xl px-2 py-2">COURSE</h4>
+                    <p>
+                        Connect with people across the globe and learn from them.
+                    </p>
+                </div>
+
+            </div>
+
+            
+                <a class="hidden md:block" href="#">
+                    <button type="button" class="my-14 hover:text-[#f28f10] underline text-lg hover:underline">
+                        SEE THE WHOLE PROGRAM
+                    </button>
+                </a>
+
+                <a class="block md:hidden" href="#">
+                    <button type="button" class="my-14 text-lg text-white bg-[#f28f10] p-4 active:border-white active:border-2 ">
+                        JOIN PROGRAM
+                    </button>
+                </a>
+            
+
+        </section>
+
+<!-- SPEAKERS -->
+
+<section class="flex justify-center items-center flex-col py-20 px-[4vw]" >
+    <h2 class="mp-txt text-2xl md:text-4xl font-semibold hover:text-[#f28f10]">
+        SPEAKERS
+    </h2>
+    <div class="mp-underline"></div>
+    <div class="grid grid-cols-1 md:grid-cols-2 justify-center mt-14" id="speaker-section">
+        <!-- SPEAKERS INJECTED -->
+    </div>
+</section>
+
+<!-- PARTNERS -->
+
+<section class="bg-[#272a31] text-white flex justify-center items-center flex-col py-10">
+    <h2 class="mp-txt text-2xl md:text-4xl font-semibold hover:text-[#f28f10]">
+        PARTNERS
+    </h2>
+
+    <div class="mp-underline"></div>
+
+    <div class="mt-14 w-full flex justify-center items-center gap-14 flex-col md:flex-row">
+
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/bmw-logo-svgrepo-com.svg" alt="bmw logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/apple-logo-svgrepo-com.svg" alt="apple logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/google-icon-logo-svgrepo-com.svg" alt="google logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/instagram-2-1-logo-svgrepo-com.svg" alt="insta logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/whatsapp-icon-logo-svgrepo-com.svg" alt="wa logo">
+
+    </div>
+
+</section>
```

Comment:
In your mobile version, this section should not appear on the main page. It appears just on the about page. Kindly fix this. :pray: :pray: 
Your design
![image](https://user-images.githubusercontent.com/98363075/203326457-98dd4042-cfee-450f-b65d-4ea9bc877f7a.png)

Template design
![image](https://user-images.githubusercontent.com/98363075/203326757-63851a0d-4cb9-4fff-ae1c-fcab206c02b2.png)

Reply:
can you please give name of section which I have to remove. Your this and that I am not able to understand. @Santiago220991